modules/detect_target/detect_target.py

```diff
@@ -17,10 +17,11 @@ class DetectTarget:
     """
     Contains the YOLOv8 model for prediction.
     """
-    def __init__(self, device: "str | int", model_path: str, save_name: str = ""):
+    def __init__(self, device: "str | int", model_path: str, enable_half: bool, save_name: str = ""):
```

Comment:
Add docstring explaining each of the parameters. I can help with that if required.